framework/src/wifi_manager/wifi_manager_state.c

```diff
@@ -682,6 +674,34 @@ wifi_manager_result_e _handler_on_scanning_state(wifimgr_msg_s *msg)
 	return wret;
 }
 
+wifi_manager_result_e _handler_get_stats(wifimgr_msg_s *msg)
+{
+	wifi_manager_result_e wret = WIFI_MANAGER_SUCCESS;
+	trwifi_stats_s stats;
+	wifi_utils_result_e res = wifi_utils_get_stats(&stats);
+	if (res != WIFI_UTILS_SUCCESS) {
+		wret = WIFI_MANAGER_FAIL;
```

Comment:
In this case, we don't need to or should not set the information from stats in line from 685 to 700.
I think, we can just return the failure.